src/doom/g_game.c

```diff
@@ -683,18 +683,42 @@ void G_BuildTiccmd (ticcmd_t* cmd, int maketic)
     {
         static unsigned int kbdlookctrl = 0;
 
-        if (gamekeydown[key_lookup] || joylook < 0)
+        if (gamekeydown[key_lookup])
         {
             look = lspeed;
             kbdlookctrl += ticdup;
         }
         else
-        if (gamekeydown[key_lookdown] || joylook > 0)
+        if (gamekeydown[key_lookdown])
         {
             look = -lspeed;
             kbdlookctrl += ticdup;
         }
         else
+        if (use_analog && joylook)
+        {
+            joylook = joylook * joystick_look_sensitivity / 10;
```

Comment:
We have the `BETWEEN` macro for this:
https://github.com/fabiangreffrath/crispy-doom/blob/5ad5f9d2c679e8e2e6341c9561377c0f09fe5256/src/crispy.h#L33